carbon-design-system/carbon
# chore(react): update storybook to use fast-sass-loader in dev

Update our sass config to use `fast-sass-loader` in development, while still using `sass-loader` in production builds. Also enables external CSS by default in prod.

#### Changelog

**New**

**Changed**

- Add copyright banners
- Update entrypoint to hooks
- Update sass strategy to use conditionally use `fast-sass-loader`

**Removed**

#### Testing / Reviewing

- Pull down PR
- Run `yarn storybook` in `packages/react`
- Change a style in a component file, like `_button.scss` and verify compilation takes <5s
- Run `yarn build-storybook` to build storybook files
- Run `npx serve storybook-static` and visit and verify project built correctly
